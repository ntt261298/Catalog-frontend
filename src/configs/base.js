export default {
  apiHost: 'http://localhost:5000',
};
