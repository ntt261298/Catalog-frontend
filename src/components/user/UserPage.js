import React from 'react';
import '../../style/user.css';
import UserItems from './UserItems';
import Header from '../layout/Header';

export const UserPage = () => (
  <div>
    <Header />
    <div className="main">
      <UserItems />
    </div>
  </div>
);

export default UserPage;
