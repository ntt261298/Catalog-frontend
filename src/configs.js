import {
  configure, shallow, mount, render,
} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

// Enzyme config
configure({ adapter: new Adapter() });

// API host config
const defaultHost = 'http://localhost:5000';

export {
  shallow, mount, render, defaultHost,
};
