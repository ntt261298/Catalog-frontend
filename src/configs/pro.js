import base from 'configs/base';

export default {
  ...base,
};
