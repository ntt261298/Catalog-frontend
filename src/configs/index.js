import {
  configure, shallow, mount, render,
} from 'enzyme/build';
import Adapter from 'enzyme-adapter-react-16/build';
import dev from 'configs/dev';
import pro from 'configs/pro';

// Enzyme config
configure({ adapter: new Adapter() });

let configs;

if (process.env.REACT_APP_ENV === 'dev') {
  configs = dev;
} else if (process.env.REACT_APP_ENV === 'pro') {
  console.log('pro');
  configs = pro;
} else {
  // Default configs is dev
  configs = dev;
}

export { shallow, mount, render };

export default configs;
