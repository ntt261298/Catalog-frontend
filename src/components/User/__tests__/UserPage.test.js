import React from 'react';
import { shallow } from 'configs';
import { UserPage } from 'components/User/UserPage';

describe('components/User/UserPage', () => {
  let wrapper;

  const setup = () => {
    wrapper = shallow(
      <UserPage />,
    );
  };

  it('should render User page', () => {
    setup();
    // Expect the wrapper object to be defined
    expect(wrapper).toMatchSnapshot();
  });
});
