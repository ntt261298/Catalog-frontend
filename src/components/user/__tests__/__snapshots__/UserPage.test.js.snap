// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/user/UserPage.js should render user page 1`] = `
<div>
  <Connect(Component) />
  <div
    className="main"
  >
    <Connect(UserItems) />
  </div>
</div>
`;
